yara-x.yaml

```diff
@@ -64,5 +63,3 @@ test:
     - runs: |
         yr --help
         yr --version | grep ${{package.version}}
-    - runs: |
-        time -v mal --min-risk critical --min-file-risk critical scan yr /usr
```

Comment:
It makes more sense to add this to the malcontent tests anyway.

Reply:
We don't automatically perform reverse-dependency testing currently: I think this test serves to ensure that the `malcontent` package won't be regressed by new `yara-x` package landing in repos.

Reply:
Right. It's just bit awkward when the version changes:
```
failed to test package: unable to build guest: unable to generate image: installing apk packages: error getting package dependencies: solving "malcontent" constraint: resolving "malcontent-1.8.6-r0.apk" deps:
solving "so:libyara_x_capi.so.0.12" constraint:   yara-x-0.12.0-r3.apk disqualified because yara-x-0.13.0-r0.apk already provides pc:yara_x_capi
  yara-x-0.12.0-r4.apk disqualified because yara-x-0.13.0-r0.apk already provides cmd:yr
  yara-x-0.12.0-r5.apk disqualified because yara-x-0.13.0-r0.apk already provides cmd:yr
```

Reply:
Re-added the test in [`e52ca72` (#41190)](https://github.com/wolfi-dev/os/pull/41190/commits/e52ca72a484e75250f0e087f5c50a6644780f566) and it will avoid any dependency resolution issues.